README.md

```diff
@@ -9,6 +9,25 @@ The official PayPal integration of Solidus.
 Getting Started
 ---------------
 
+Whatever the setup you'll choose, you'll need to declare your PayPal Partner-ID and Partner-Client-ID as environment
+variables or by setting their values directly on `SolidusPaypalCommercePlatform` first:
+
+Set the required ENV variables before running your specs:
```

Comment:
This and the below lines appear to repeated in the `testing` section below. Looks like that's where they were meant to go.

Reply:
That's necessary for any environment so if we want to just cite them once it's probably better to do so in "Getting Started"